Formula/g/grails.rb

```diff
@@ -16,8 +16,17 @@ class Grails < Formula
 
   depends_on "openjdk@11"
 
+  resource "cli" do
+    url "https://github.com/grails/grails-forge/releases/download/v6.2.0/grails-cli-6.2.0.zip"
+    sha256 "de6eaa4389ce4cb08081e219f8838b6cb1a0445c8e6a4dd66cb4cc2fa7652776"
+  end
+
   def install
+    odie "cli resource needs to be updated" if version != resource("cli").version
+
+    resource("cli").stage { libexec.install Dir["*"] }
     rm_f Dir["bin/*.bat", "bin/cygrails", "*.bat"]
+
     libexec.install Dir["*"]
     bin.install Dir[libexec/"bin/*"]
```

Comment:
Given this:

```console
$ ls grails-cli-6.2.0
LICENSE bin     lib
$ ls grails-cli-6.2.0/bin
grails            grails.bat        grails_completion
$ ls grails-6.2.0
INSTALL LICENSE dist    lib     media   src
```

We should:

1. Remove `grails.bat`.
2. Install `grails_completion` appropriately.
3. Clean up existing unused logic.

```suggestion
    libexec.install Dir["*"]

    resource("cli").stage do
      rm_f "bin/grails.bat"
      (libexec/"lib").install Dir["lib/*.jar"]
      bin.install "bin/grails"
      bash_completion.install "bin/grails_completion" => "grails"
    end

```